ambry-api/src/main/java/com/github/ambry/config/ServerConfig.java

```diff
@@ -112,6 +112,12 @@ public class ServerConfig {
   @Default("com.github.ambry.server.AmbryServerSecurityServiceFactory")
   public final String serverSecurityServiceFactory;
 
+  /**
+   * The information of the RepairRequestsDB to connect with.
+   */
+  @Config("server.repair.requests.db")
+  public final String dbInfo;
+
```

Comment:
And after that we can create a separate config class for RepairRequestsDBConfig.

Reply:
Added RepairRequestsDbConfig for the MysqlRepairRequestsDbFactory.